build/cygnus.js

```diff
@@ -1,29 +1,28 @@
-var _this2 = this,
-    _arguments = arguments;
-
 /* global cygnus, Blob, Worker, XMLHttpRequest */
 
+const _arguments = arguments;
+
 module.exports = {
   supportsHistory: !!window.history,
   supportsWorkers: !!window.Worker,
   supportsPromises: !!Promise,
   ready: false,
   pages: {},
-  init: () => {
+  init: function () {
```

Comment:
That's what I thought. I had already changed to a `function` declaration my end to get this working.